eox_tenant/apps.py

```diff
@@ -49,7 +49,24 @@ class EdunextOpenedxExtensionsTenantConfig(AppConfig):
                         'signal_path': 'celery.signals.task_prerun',
                     },
                 ],
-            }
+            },
+            'cms.djangoapp': {
+                'relative_path': 'signals',
+                'receivers': [
+                    {
+                        'receiver_func_name': 'start_tenant',
+                        'signal_path': 'django.core.signals.request_started',
+                    },
+                    {
+                        'receiver_func_name': 'finish_tenant',
```

Comment:
we never use this signal in the lms. Should we add it here?

Reply:
I don't remember the reason to add this in lms, but if we don't want to add that here, because we don't use it, we should consider to clean all the unused code.

Reply:
I agree. At least lets clean up this 2 signals from this and also the lms